go-update/src/main/java/org/reactome/release/goupdate/GoLineProcessor.java

```diff
@@ -30,7 +30,7 @@ private GoLineProcessor() {}
 	 * @param currentGOID - The ID of the GO term currently being processed, line by line.
 	 * @return The GO Accession of the GO term currently being processed. Will be different from <code>currentGOID</code> if the ID for a new GO term is seen on this <code>line</code>
 	 */
-	static String processLine(String line, String currentGOID, Map<String, Map<String,Object>> goTerms)
+	static String processLine(String line, String currentGOID, Map<String, Map<String, Object>> goTerms)
```

Comment:
Should the param annotations for goTerms and currentGOID be switched so they are in the same order as the method signature?  Also, it'd be good for the description to elaborate a bit about what the map of maps hold in terms of their keys and values.